packages/fxa-settings/e2e/tools/firefox_profile_creator.js

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var UA_OVERRIDE =
+  'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:40.0) Gecko/20100101 Firefox/40.0 FxATester/1.0';
+
+var FirefoxProfile = require('firefox-profile');
+var myProfile = new FirefoxProfile();
+var profile = null;
+
+if (process.argv.length > 1) {
+  try {
+    profile = JSON.parse(process.argv[2]);
+  } catch (e) {
+    throw new Error('Failed to parse args');
+  }
+}
+
+if (profile) {
+  if (profile.isTestingPairing) {
```

Comment:
Should we typescript-ify all/any of these files? Even small wins like `if (profile?.isTestingPairing) {` are nice (albeit small) wins.